<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="youtube_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string>From Youtube</string>
   </property>
  </widget>
  <widget class="QPushButton" name="file_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>201</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string>From file</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
